src/modules/profile/profile.service.ts

```diff
@@ -35,19 +39,35 @@ export class ProfileService {
     return allProfile;
   }
 
-  async updateSingleProfile(id: number, save: { saved: boolean }) {
-    const { saved } = save;
-    if (id && save) {
-      await this.profileRepository.update({ id: id }, { saved: saved });
+  async updateSingleProfile(id: number, save: SavedProfileDto): Promise<SaveFreelancerEntity[] | SavedProfileDto> {
+    const { saved, clientId } = save;
+    const profile = await this.saveProfileFreelancerRepository
+      .createQueryBuilder('Freelancer')
```

Comment:
please use lowercase instead of Upper case for each QueryBuilder

Reply:
fixed it